byway/src/main/java/com/google/sps/servlets/StopsServlet.java

```diff
@@ -26,26 +36,35 @@
 /** Servlet that handles data for stops */
 @WebServlet("/api/stop")
 public final class StopsServlet extends HttpServlet {
-  @GuardedBy("this")
-  private ArrayList<String> stops = new ArrayList<String>();
-
   private final Gson gson = new Gson();
-  // Implement datastore in routepage branch
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static final Type ARRAYLIST_STRING = new TypeToken<ArrayList<String>>() {}.getType();
+  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
```

Comment:
Try to keep static fields separate from non-static fields. Usually it's good to put statics first in the class (since they're shared across all instances, it's nice to show them first) and put a blank line after them to separate them from non-static field.s

Reply:
Oh okay, thanks for pointing this out I just lumped all final fields together. Changed.